### PR TITLE
File structure changes

### PR DESCRIPTION
**Pull request by harshit-sharma-gits**
Did some changes in the file structure to make contributions, as well as maintaining easy.
Also raised issue #7 with clear instructions on how to create your portfolio (or say, contribute)

I'd also mention, some changes must be done in the README.md so that the issue #7 and README.md don't clash on instructions.

### CODE REVIEW

File: Your Portfolio/Chickisnak.html [hunk 1]
```diff
@@ -505,7 +505,7 @@ <h1 class="m-0 display-5"><span class="text-primary">Akhand </span>Pratap Singh<
                     <!-- TRY TO KEEP YOUR IMAGE IN 1:1,THAT IS, SQUARE -->
 
                     <img class="img-fluid w-100 rounded-circle shadow-sm" 
-                    src="210111005.jpeg"
+                    src="Images/Chickisnak.jpeg"
```
Comment (aastha-chauhan-gits) [inline Your Portfolio/Chickisnak.html:508]: Can you make this change in the template as well?

**Review by aastha-chauhan-gits — CHANGES_REQUESTED** [review]
I've added a comment please check it

File: Your Portfolio/Chickisnak.html (hunk 1 above)
Comment (harshit-sharma-gits) [reply Your Portfolio/Chickisnak.html:508]: I've made the requested changes, please check them out

**Review by isyneha — APPROVED** [review]
It's good, I was also thinking of optimising it that way.

**Pull request merged**